yarn.lock

```diff
@@ -2358,6 +2358,25 @@
     lodash.isequal "^4.5.0"
     qs "^6.9.1"
 
+"@opentripplanner/core-utils@^8.2.1":
```

Comment:
Can you run `yarn-deduplicate` so it combines all the 8.x core-utils references?